types/glob-stream/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "@types/glob": "~7.2.0"
+        "glob": "^10.2.1"
```

Comment:
Is this the first version that ships its own types?

Reply:
Iirc, the first version to introduce it's own types was 9.0.0

Reply:
Since glob-stream doesn’t depend on glob, but rather the types just need the types, shouldn’t the version string allow glob 9.x if it works? Maybe `^9.0.0 || ^10.0.0` or something?

Reply:
Sounds great - will do asap

Reply:
> Since glob-stream doesn’t depend on glob, but rather the types just need the types, shouldn’t the version string allow glob 9.x if it works? Maybe `^9.0.0 || ^10.0.0` or something?

Gave it a go - sadly, `^9.0.0` won't work as all versions of `glob@9` use `minipass` in a way that all TypeScript projects depending on it are forced to have `esModuleInterop` enabled. Iirc, `esModuleInterop` was some no-go, right?

Reply:
I feel like we might be able to sidestep this whole thing by updating `glob-stream`'s types; the latest version (v8) no longer uses `glob`, but instead `anymatch` (and transitively `picomatch`), so the one place `glob`'s types are used here for an options bag could be changed to `picomatch`'s.

That is of course assuming that 1) leaving this type as-is for people to use will still be fine, and 2) someone is willing to make that change.

Reply:
Just to be super clear, I am suggesting going right to `picomatch` for the types, as `anymatch` has unfortunately decided to hand-write its types and their options bag is incorrect.

Reply:
Thanks a lot for poonting this out!
I'll keep that in mind

Reply:
Of course, maybe the fix is just to also fix anymatch but I think we are trying to be expedient here :D